frontend/src/components/dialogs/SecretDialog.vue

```diff
@@ -198,6 +208,14 @@ export default {
     shootsByInfrastructureSecret () {
       const name = get(this.secret, 'metadata.name')
       return filter(this.shootList, ['spec.secretBindingName', name])
+    },
+    helpStyle () {
```

Comment:
Use classes if possible. Static attributes should be static not bound. Use Object Syntax (inline if possible) not String syntax https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1

Reply:
done